app/view_models/workarea/storefront/blog_view_model.rb

```diff
@@ -68,7 +68,7 @@ def ordered_entries
       end
 
       def scoped_entries_from_database
-        scope = ordered_entries.active
+        scope = ordered_entries.select(&:active?)
```

Comment:
I don't think this can work? Using `select(&:active?)` will change `scope` into an array. Maybe there's no test coverage on this? 😮

Reply:
yeah i think the `.tagged_with` and `.all_tags` methods are gonna give us some shit here...